Cargo.lock

```diff
@@ -19,30 +19,30 @@ checksum = "f26201604c87b1e01bd3d98f8d5d9a8fcbb815e8cedb41ffccbeb4bf593a35fe"
 
 [[package]]
 name = "aead"
-version = "0.4.3"
```

Comment:
There shouldn't be so many changes here. I need to revert the `wasi-crypto` git sub module again back to `main` so these get removed.